packages/flutter/test/material/navigation_rail_test.dart

```diff
@@ -3631,6 +3632,96 @@ void main() {
     expect(tester.takeException(), isNull);
   });
 
+  testWidgets('NavigationRail labels shall not overflow if longer texts provided - extended', (WidgetTester tester) async {
+
+    // Adding widget separately to ensure the navigation rail's width is fixed so navigation destination shall wrap itself as expected
```

Comment:
I'm not sure what this comment is meant for. Maybe this?
```suggestion
    // The navigation rail has a narrow width constraint. It should wrap itself.
```